### PR TITLE
Added support for reading Deck data

### PR DESCRIPTION
**Pull request by JensGM**

### CODE REVIEW

**Comment by joakim-hove** [comment]
git gripe: Instead of merging in the latest changes from upstream with a variety of `git merge upstream/master` - please rebase instead - then the PR becomes cleaner for the reviewer. Starting on your feature branch - with no uncomitted changes;

```txt
git fetch upstream
git rebase upstream/master
git push -f origin $your_branch
```

**Comment by joakim-hove** [comment]
OK - this looks OK; but we need Python test(s) actually accessing the data. 

<Disclaimer>I have not really looked into this</Disclaimer> For the double data there are two accessors in C++ - `getRawDoubleData( )` which returns the unmodified values as read from the input file, and `getSIDoubleData( )` - I would assume/fear that some of that complexity must be exposed in sunbeam.

**Comment by JensGM** [comment]
@joakim-hove I have now tried to address the issues you raised.

**Comment by pgdr** [comment]
Wouldn't it make more sense if `DeckItem.data` was a property that returned the correct list.

And not exposing `int_data`, `string_data`  etc?

**Comment by JensGM** [comment]
Commited a new variant of this PR where deck items are converted to lists. This does means that the SI version of the double data is not available. But because this is in the context of the Deck data structure (which is just a representation of the text file), I don't think accessing the unit converted values are very interesting. At the very least not if you are to write the deck back out to a file.

**Comment by joakim-hove** [comment]
Test!

**Comment by JensGM** [comment]
Added test, found bugs, fixed bugs.

File: tests/deck.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+REGIONS
+OPERNUM
+3 3 1 2 /
+FIPNUM
+1 1 2 3 /
+"""
+
+    def setUp(self):
+        self.spe3fn = 'spe3/SPE3CASE1.DATA'
+        self.norne_fname = os.path.abspath('../../examples/data/norne/NORNE_ATW2013.DATA')
```
Comment (pgdr) [inline tests/deck.py]: Perhaps you could 
```python
from os.path import abspath, join
root = join(__file__, '..', '..',)
self.norne_fname = join(root, 'examples', 'data', 'norne', 'NORNE_ATW2013.DATA') 
```

File: tests/deck.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        self.deck = sunbeam.parse_deck(self.DECK_STRING)
+
+    def testDeckIn(self):
```
Comment (pgdr) [inline tests/deck.py]: Please only snake_case in Python function/method names.

**Pull request merged**